<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui</class>
 <widget class="QMainWindow" name="gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>神龙翻译 v1.8</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>by abcdesteve</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>400</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>神龙翻译 v1.8</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>输入要翻译的内容</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="topmost">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>窗口置顶</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="file_select">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>文件翻译</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>翻译结果</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="hide_char">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>忽略标点</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="web">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>打开网页</string>
    </property>
   </widget>
   <zorder>name_label</zorder>
   <zorder>title_label</zorder>
   <zorder>output</zorder>
   <zorder>input</zorder>
   <zorder>hide_char</zorder>
   <zorder>file_select</zorder>
   <zorder>topmost</zorder>
   <zorder>web</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
